docs/build/index.mdx

```diff
@@ -6,6 +6,12 @@ description: Build circuits with Qiskit and Qiskit Runtime
 
 # Introduction
 
-In the build phase, you create quantum programs that represent the problem you are solving. The foundation of quantum programs are quantum circuits, which are made up of gates (or operations) that are used to manipulate qubits in the quantum computer.  Depending on your needs, you can use Qiskit or OpenQASM to create your circuits. 
+In the build phase, you create quantum programs that represent the problem you are solving. The foundation of quantum programs are quantum circuits, which are made up of operations - including gates, measurement, and reset of qubits - that manipulate qubits in the quantum computer.  Depending on your needs, you can use Qiskit or OpenQASM to create your circuits. 
 
-This documentation covers building circuits and operators in Qiskit, visualizing circuits, and other circuit-building tools.
\ No newline at end of file
+All tasks require building one or more [quantum circuits](circuit-construction). Some tasks additionally require constructing [quantum operators](operators_overview) to define properties of quantum states that we want to estimate or measure.
+
+Qiskit enables working with circuits (and, to a lesser extent, operators) at various abstraction levels: abstract, virtual, physical, scheduled, and pulse programs.
+
+At the most abstract level, we have a task-oriented lens in the [circuit library](circuit-library). We can also express operations in abstract mathematical terms using operators, isometries, and classical/Boolean functions. Then we have virtual circuits where those mathematical abstractions take on a concrete representation in terms of a concrete gate set. At the physical level those instructions are mapped to specific physical qubits and instructions are re-written to reflect the connectivity and native gate set of a target hardware platform. Scheduled circuits introduce timing information. Finally, [pulse programs](pulse) represent signals on channels.
+
+Qiskit and [OpenQASM](introduction-to-qasm) further support the notion of extended circuits, which expand the set of allowed operations to include real-time computations on classical values. Qiskit's tooling for working with this richer family of circuits is found in the section on [Classical feedforward and control flow](classical-feedforward-and-control-flow).
```

Comment:
@javabster - any thoughts on whether the links in this section look ok, or whether it would be better to leave the links out entirely and list them separately at the bottom (or not at all, since they are also in the left nav)?

Reply:
Right now the intro section doesn't mention the bit-ordering or visualization pages, but maybe that's also ok